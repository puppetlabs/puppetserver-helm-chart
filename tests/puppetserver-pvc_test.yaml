suite: test puppet puppet volume creation
templates:
  - puppetserver-pvc.yaml
release:
  name: puppetserver
  namespace: puppet
tests:
  - it: should not create the volume if "customPersistentVolumeClaim.puppet.enable" is true
    set:
      puppetserver.customPersistentVolumeClaim.puppet.enable: true
    asserts:
      - hasDocuments:
          count: 0
  - it: should create the volume by default
    asserts:
      - isKind:
          of: PersistentVolumeClaim
