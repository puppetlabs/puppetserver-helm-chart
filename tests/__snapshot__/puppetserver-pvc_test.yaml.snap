manifest should match snapshot:
  1: |
    apiVersion: v1
    kind: PersistentVolumeClaim
    metadata:
      annotations: null
      labels:
        app.kubernetes.io/component: puppetserver
        app.kubernetes.io/instance: puppetserver
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: puppetserver
        app.kubernetes.io/version: 7.13.0
        helm.sh/chart: puppetserver-9.3.2
      name: puppetserver-puppet-claim
    spec:
      accessModes:
        - ReadWriteOnce
      resources:
        requests:
          storage: 400Mi
